partner_firstname/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     'name': 'Partner first name and last name',
     'summary': "Split first name and last name for non company partners",
-    'version': '10.0.2.0.0',
+    'version': '10.0.2.0.1',
```

Comment:
Version should be 10.0.2.1.0

Reply:
Hmmmm I would consider this more of a fix, but no prob

Reply:
But it requires an update, isn't it?